src/pe.cc

```diff
@@ -12,56 +12,56 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "bloaty.h"
 #include "absl/strings/substitute.h"
+#include "bloaty.h"
 #include "util.h"
 
 using absl::string_view;
+using std::string;
 
 namespace bloaty {
 namespace pe {
-const uint16_t dos_magic = 0x5A4D;  // MZ
+constexpr uint16_t dos_magic = 0x5A4D;  // MZ
 
+// From LIEF/include/LIEF/PE/Structures.hpp.in
 //! Sizes in bytes of various things in the COFF format.
-enum class StructSizes {
-  kHeader16Size = 20,
-  kHeader32Size = 56,
-  kNameSize = 8,
-  kSymbol16Size = 18,
-  kSymbol32Size = 20,
-  kSectionSize = 40,
-  kRelocationSize = 10,
-  kBaseRelocationBlockSize = 8,
-  kImportDirectoryTableEntrySize = 20,
-  kResourceDirectoryTableSize = 16,
-  kResourceDirectoryEntriesSize = 8,
-  kResourceDataEntrySize = 16,
-};
-
+constexpr size_t kHeader16Size = 20;
+constexpr size_t kHeader32Size = 56;
+constexpr size_t kNameSize = 8;
+constexpr size_t kSymbol16Size = 18;
+constexpr size_t kSymbol32Size = 20;
+constexpr size_t kSectionSize = 40;
+constexpr size_t kRelocationSize = 10;
+constexpr size_t kBaseRelocationBlockSize = 8;
+constexpr size_t kImportDirectoryTableEntrySize = 20;
+constexpr size_t kResourceDirectoryTableSize = 16;
+constexpr size_t kResourceDirectoryEntriesSize = 8;
+constexpr size_t kResourceDataEntrySize = 16;
```

Comment:
@haberman I hope you don't mind, I changed these as well.

Reply:
Yep np!